app/uk.gov.hmrc.helptosaveproxy/connectors/NSIConnector.scala

```diff
@@ -116,7 +121,7 @@ class NSIConnectorImpl @Inject() (auditConnector:    AuditConnector,
     val timeContext: Timer.Context = metrics.nsiUpdateEmailTimer.time()
     val correlationId = getCorrelationId
 
-    httpProxy.put(nsiCreateAccountUrl, userInfo, true, Map(nsiAuthHeaderKey → nsiBasicAuth))(nsiUserInfoFormat, hc.copy(authorization = None), ec)
```

Comment:
removed the `needsAuditing` boolean field from http client.  because we already do an explicit event after successful email update, so the implicit auditing no longer required

Reply:
I think it might still be worth having implicit auditing in place - we log an explicit event only when the calls are successful. Also, I think TxM's expectations are that an implicit event will be there so they might be essential to whatever machine learning algorithms they have in place to look at these events

Reply:
did we not? We should probably put that back in. Are there any other cases where we've turned off implicit auditing? Perhaps it's a question worth asking TxM - if we have an explicit event logged as a result of a call do we need an implicit auditing around that call?

Reply:
sorry, i was wrong, there is implicit auditing happening for create-account